src/css/ui-design.scss

```diff
@@ -1,29 +1,31 @@
-@import 'node_modules/bootstrap/scss/bootstrap';
-@import 'variables';
-@import 'mixins';
+/*
+ * @forgerock/javascript-sdk-ui
+ *
+ * _ui-design.scss
+ *
+ * Copyright (c) 2020-2021 ForgeRock. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
 
-@import 'alert';
-@import 'animations';
-@import 'body';
-@import 'button-group';
-@import 'buttons';
-@import 'card';
-@import 'color';
-@import 'dropdown';
-@import 'forms';
-@import 'functions';
-@import 'icons';
-@import 'input';
-@import 'media';
-@import 'nav';
-@import 'navbar';
-@import 'placeholders';
-@import 'position';
-@import 'select';
-@import 'sidebar';
-@import 'spinners';
-@import 'sprite';
-@import 'switch';
-@import 'tables';
-@import 'tabs';
-@import 'typography';
```

Comment:
These were all the ui-design library components. The Bootstrap components are now being used directly.